web_src/css/repo/header.css

```diff
@@ -1,4 +1,8 @@
-.header-wrapper .fork-flag {
+.repository .secondary-nav {
+  padding-top: 12px;
+}
+
+.repository .secondary-nav .fork-flag {
```

Comment:
Why this change?

Reply:
Hmm yes we could remove the `.repository` prefix.

Reply:
Done in https://github.com/go-gitea/gitea/pull/29982/commits/44585322d49477810d30ea3d289d4d6b706cec2b.

Reply:
I mean ... it should remove `.secondary-nav` and keep it as before?

Reply:
Previously it was `.header-wrapper` and that class is replaced with `.secondary-nav` so it's a 1:1 class replacement in the selector.